packages/mobile/src/escrow/saga.ts

```diff
@@ -117,6 +220,182 @@ function* registerStandbyTransaction(context: TransactionContext, value: string,
   )
 }
 
+async function formEscrowWithdrawAndTransferTxWithNoCode(
+  contractKit: ContractKit,
+  escrowWrapper: EscrowWrapper,
+  stableTokenWrapper: StableTokenWrapper,
+  paymentId: string,
+  privateKey: string,
+  walletAddress: string,
+  metaTxWalletAddress: string,
+  value: BigNumber
+) {
+  const msgHash = contractKit.web3.utils.soliditySha3({
+    type: 'address',
+    value: metaTxWalletAddress,
+  })
+
+  const signature: string = (await contractKit.web3.eth.accounts.sign(msgHash, privateKey))
```

Comment:
The object returned by this function has the following fields: 
```
{ message, messageHash, v, r, s, signature }
```

Why do we only take the `signature` field, just to split it up into r, s, and v? Suggest we just read those directly

Reply:
Good call. It was a blind copy/paste.